### ISSUE TITLE
How can I set it to download youtube Chinese subtitles at the same time?

### ISSUE THREAD

**Issue by 911tj** [Maxstupo/ydl-ui#82]
Strongly request this feature to be added, thanks

**Comment by Maxstupo** [comment]
Currently, YDL-UI doesn't support downloading subtitles of a specific language.

It's something I plan on adding eventually.

**Comment by smithmarci** [comment]
Would it be possible to use '--all-subs' by default instead of (/along with?) '--write-sub', if "Write subtitle" option is checked in the 'Add Download...' panel? With this change all the available subtitles could be downloaded in one go, and would also make the listing of the available subs and selecting the desired subtitles unnecessary - just until this feature will be added, of course.

**Comment by Maxstupo** [comment]
It would be a good compromise for the time being. I could implement the `--sub-lang` flag, but it would require you to manually enter the language tag(s).

Something like this... 
![](https://i.gyazo.com/8a2f3fdeac42f9f20110dbaaf290ced2.png)

- Default: `--write-sub`
- Languages: `--write-sub --sub-lang ...`
- All Subs: `--all-subs`

**Comment by smithmarci** [comment]
That would be great, if this compromise solution could happen before a more complex implementation. Thanks for considering this option!

As for the suggested future implementation, when someone is looking for an exact type of subtitle they probably know their own country code, so - in my opinion - your shown above solution would be satisfactory in all respects.

**Comment by smithmarci** [comment]
Wow! It works like a charm. Thank you for adding the feature to this superb application!

However, right during my first attempt I ran into an error - although, it was a human error. I presumed that one should use the languages' ISO Alpha-2 code in uppercase capital letters. I've mistaken.

If I may suggest: the use of language codes in lowercase letters could be mentioned in the tooltips, so that no one else would make the same mistake that I did. Naturally, there are exceptions, dialects like 'fr-CA', 'es-MX', 'zh-TW',... (and many more, of course),  but maybe it could be useful to draw attention to these exceptions as well.
